scripts/zkai_d64_commitment_consistency_method_probe.py

```
@@ -0,0 +1,692 @@
+def _git_commit() -> str:
+    for env_name in GIT_COMMIT_OVERRIDE_ENVS:
+        override = os.environ.get(env_name)
+        if override and override.strip():
+            return override.strip().lower()
+    try:
+        if _dirty_worktree_paths():
+            return DIRTY_GIT_COMMIT
+    except CommitmentConsistencyProbeError:
+        return "unavailable"
+    try:
+        completed = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)